LICENSE-binary

```diff
@@ -489,11 +489,10 @@ org.checkerframework:checker-qual:2.5.2
 org.codehaus.mojo:animal-sniffer-annotations:1.21
 org.jruby.jcodings:jcodings:1.0.13
 org.jruby.joni:joni:2.1.2
-org.slf4j:jul-to-slf4j:jar:1.7.25
-org.ojalgo:ojalgo:43.0:compile
-org.slf4j:jul-to-slf4j:1.7.25
-org.slf4j:slf4j-api:1.7.25
-org.slf4j:slf4j-log4j12:1.7.25
+org.ojalgo:ojalgo:43.0
+org.slf4j:jul-to-slf4j:1.7.35
+org.slf4j:slf4j-api:1.7.35
+org.slf4j:slf4j-reload4j:1.7.35
```

Comment:
should be 1.7.36 instead of 1.7.35? @steveloughran 

```
$ find hadoop-dist/target/hadoop-3.5.0-SNAPSHOT -name '*.jar' | grep slf4j
hadoop-dist/target/hadoop-3.5.0-SNAPSHOT/share/hadoop/common/lib/slf4j-api-1.7.36.jar
hadoop-dist/target/hadoop-3.5.0-SNAPSHOT/share/hadoop/common/lib/slf4j-reload4j-1.7.36.jar
hadoop-dist/target/hadoop-3.5.0-SNAPSHOT/share/hadoop/common/lib/jul-to-slf4j-1.7.36.jar
```

Reply:
well spotted! i just pulled in the branch-3.3 license changes; looks like it is out of date there!